static/css/cf-enhancements.less

```diff
@@ -660,3 +660,34 @@ tbody {
     name: EOF
     eof: true
 */
+
+
+/* topdoc
+  name: Full Width Buttons
+  family: cf-enhancements
+  notes:
+    - "Modifier to buttons full width at small screen sizes"
+  patterns:
+    - name: Default example
+      markup: |
+        <btn class="btn btn__full">Button Text</btn>
+  tags:
+    - cf-buttons
```

Comment:
I'm not clear on what the `tags` should include and how they differ from `family`, but most existing topdocs include the family name as a tag, so this should be `cf-enhancements`.